provider/cluster/service.go

```diff
@@ -360,37 +359,11 @@ func (s *service) teardownLease(lid mtypes.LeaseID) {
 	}
 }
 
-func findDeployments(ctx context.Context, log log.Logger, client Client, session session.Session) ([]ctypes.Deployment, error) {
+func findDeployments(ctx context.Context, log log.Logger, client Client, _ session.Session) ([]ctypes.Deployment, error) {
```

Comment:
Can't we just remove this method?

Reply:
yeah

Reply:
I kept it as a placeholder for something that looks in CRDs/etcd but I can remove it.

Reply:
OK, I think the same goes for the other method below